features/step_definitions/payment_steps.rb

```diff
@@ -28,7 +28,7 @@
   Capybara::Screenshot.screenshot_and_open_image
 end
 
-Then(/^I should see "([^"]*)" on the order confirmation$/) do |content|
-  sleep(4)
+Then(/^I should see "([^"]*)" on the order confirmation page$/) do |content|
+  sleep(2)
```

Comment:
Let me check. The problem is that there is a small animation when you click submit on the stripe form. That take some time until you are actually on the order confirmation page. That's why we put a sleep there to make it go green before.